src/osip-monitoring-controller/src/main/resources/Bundle.properties

```diff
@@ -19,3 +19,23 @@ monitoring.tank.progress.fillLevel=Fill level
 monitoring.tank.progress.temperature=Temperature
 monitoring.mixTank.motorSpeed=Motor speed
 monitoring.mixTank.color=Color
+monitoring.helpdialog.title=Help
+monitoring.helpdialog.mainTab.header=Shell
+monitoring.helpdialog.mainTab.introduction=The OSIP monitoring shell can be used to monitor the tanks, pipes and alarms of the production site simulation via the OPC UA protocol.
+monitoring.helpdialog.mainTab.layoutSection=Shell Layout
+monitoring.helpdialog.mainTab.layoutText=The layout of the monitoring shell can be divided into five parts. At first there are the information about the upper tanks of the production site. These boxes are located at the upper side of the shell and are bordered in the color of the liquid in the tank. At the right of those parts, there is the information about the mix tank. The color is shown within the box. Underneath those boxes there is a textfield which is used to print alarm messages.
+monitoring.helpdialog.mainTab.alarmSection=Alarms
+monitoring.helpdialog.mainTab.alarmText=Every tank box has four possible alarms which are shown as small red, green or grey circles at the upper left corner of the box. If the circle is green the alarm won't be active, if it is red then the alarm will be active. In case, the circle is gray, the alarm is disabled. Additionally there is a big red and green circle at the bottom right of the shell. If the red circle is filled, some alarm will be active and if the green circle is filled, no alarm will be active. When both circles are gray, every alarm will be disabled.
+monitoring.helpdialog.mainTab.gaugeSection=Gauges
+monitoring.helpdialog.mainTab.gaugeText=Every tank box has four gauges showing various properties of the tank. The gauge at the upper left side shows the input flow rate of liquid flowing into the tank. The gauge at the upper right side shows the output flow rate of liquid flowing out of the tank. Below is a fill rate gauge and a temperature scale.
+monitoring.helpdialog.mainTab.progressSection=Progress diagram
+monitoring.helpdialog.mainTab.progressText=Below the gauges are two progress diagrams, accessible via two tabs, which show the progress of the fill level and the temperature. This can be used to analyze the history of the fill level and temperature changes.
+monitoring.helpdialog.settingsTab.header=Settings
+monitoring.helpdialog.settingsTab.introduction=The settings can be opened by clicking on the "File" menu and selecting the menu entry "Settings".
+monitoring.helpdialog.settingsTab.generalSection=General Settings
+monitoring.helpdialog.settingsTab.generalText=The general settings tab can be used to configure the update interval which is the frequency used by the monitoring shell to request data from the production site simulation. Additionally one can specify the hostname or IP-address of the production site simulation by filling it into the "Server-Hostname" field. Below this field one can select the IP ports for every tank.
```

Comment:
Additionally **[Komma]** one can specify the hostname or IP-address of the production site simulation by filling it into the "Server-Hostname" field. Below this field **[Komma]** one can select the IP ports for every tank.